### PR TITLE
fix(query): fix queries expected value to 'should be...'

### PR DESCRIPTION
**Pull request by liorj-orca**
Closes #

**Proposed Changes**
-
-
-

I submit this contribution under the Apache-2.0 license.

### CODE REVIEW

File: assets/queries/terraform/aws/cloudtrail_multi_region_disabled/query.rego [hunk 1]
```diff
@@ -56,7 +56,7 @@ CxPolicy[result] {
 		"searchKey": sprintf("aws_cloudtrail[%s].include_global_service_events", [name]),
 		"searchLine": common_lib.build_search_line(["resource", "aws_cloudtrail", name, "include_global_service_events"], []),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": sprintf("aws_cloudtrail[%s].include_global_service_events undefined or is set to true", [name]),
+		"keyExpectedValue": sprintf("aws_cloudtrail[%s].include_global_service_events undefined or should be set to true", [name]),
```
Comment (rafaela-soares) [inline assets/queries/terraform/aws/cloudtrail_multi_region_disabled/query.rego:59]:
```suggestion
                "keyExpectedValue": sprintf("aws_cloudtrail[%s].include_global_service_events should be undefined or set to true", [name]),
```

File: assets/queries/googleDeploymentManager/cluster_master_authentication_disabled/query.rego [hunk 2]
```diff
@@ -52,8 +52,8 @@ CxPolicy[result] {
 		"resourceName": resource.name,
 		"searchKey": sprintf("resources.name={{%s}}.properties.masterAuth", [resource.name]),
 		"issueType": "IncorrectValue",
-		"keyExpectedValue": "Attribute 'masterAuth.username' to be not empty and attribute 'masterAuth.password' to be not empty",
-		"keyActualValue": "Attribute 'masterAuth.username' is empty or attribute 'masterAuth.password' is empty", 
+		"keyExpectedValue": "Attribute 'masterAuth.username' should be not empty and attribute 'masterAuth.password' should be not empty",
```
Comment (rafaela-soares) [inline assets/queries/googleDeploymentManager/cluster_master_authentication_disabled/query.rego:55]:
```suggestion
                "keyExpectedValue": "Attribute 'masterAuth.username' should not be empty and attribute 'masterAuth.password' should be not empty",
```

**Review by rafaela-soares — CHANGES_REQUESTED** [review]
Hello, @liorj-orca!

Thank you so much for continuing to improve KICS queries 🚀 

I only have minor suggestions:

File: assets/queries/terraform/aws/cloudtrail_multi_region_disabled/query.rego (hunk 1 above)
Comment (liorj-orca) [reply assets/queries/terraform/aws/cloudtrail_multi_region_disabled/query.rego:59]: will be fixed right away :)

**Review by rafaela-soares — APPROVED** [review]
LGTM 🚀 

Thank you so much, @liorj-orca!

**Pull request merged**